### PR TITLE
Python: Add full-path modeling of Django more view classes

### PR DESCRIPTION
**Pull request by RasmusWL**
and a slightly concerning bugfix in `django_attr` :grimacing:

### CODE REVIEW

File: python/ql/test/experimental/library-tests/frameworks/django-v2-v3/routing_test.py [hunk 1]
```diff
@@ -36,6 +37,12 @@ def get(self, request, untrusted):  # $ requestHandler routedParameter=untrusted
         return HttpResponse('ClassView get: {}'.format(untrusted))  # $HttpResponse
 
 
+# direct import with full path to `View` class (previously not supported)
```
Comment (yoff) [inline python/ql/test/experimental/library-tests/frameworks/django-v2-v3/routing_test.py:40]:
```suggestion
# direct import with full path to `View` class
```
Let us keep the development history out of the comments :-)

**Review by yoff — CHANGES_REQUESTED** [review]
Looks great (you seem to have modelled much more than we test). One nitpick.

File: python/ql/test/experimental/library-tests/frameworks/django-v2-v3/routing_test.py (hunk 1 above)
Comment (RasmusWL) [reply python/ql/test/experimental/library-tests/frameworks/django-v2-v3/routing_test.py:40]: In general I would agree with this. But in this specific instance, I wanted to highlight why we have both `ClassView` and `ClassView2`. I guess the other part of the comment is enough

Comment (RasmusWL) [reply python/ql/test/experimental/library-tests/frameworks/django-v2-v3/routing_test.py]: Merged the suggestion (but kept it open so you can see my response)

**Review by yoff — APPROVED** [review]
LGTM, happy to merge once the tests pass

**Pull request merged**